demography.md

```diff
@@ -15,7 +15,7 @@ kernelspec:
 
 **Georgia Tsambos**
 
-So far, we've been simulating samples from a single population of a constant size,
+By default, {func}`msprime.sim_ancestry` simulates samples from a single population of a constant size,
 which isn't particularly exciting!
 One of the strengths of msprime is that it can be used to specify quite complicated
 models of demography and population history with a simple Python API.
```

Comment:
In the next cell we can delete the %config stuff. We could consider hiding this cell too (:tags: [hide-input]) or putting the imports in with the first cell we show?

Reply:
I'd actually vote to keep it in as a separate cell, and showing it quite prominently outside of the nested headings, to reinforce to any readers who aren't familiar with python/jupyter that this is *all* they need to run all of the examples on the page.

Reply:
OK, sounds good.